index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Awesome Books Home Page</title>
+ <link rel="stylesheet" href="/Assets/css/styles.css">
```

Comment:
- [ ] I think you have connected the CSS with wrong path. (Hint: It should be `./styles.css`). :smile:

Reply:
done